tests/acceptance/test_primer.py

```diff
@@ -0,0 +1,53 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+import contextlib
+import os
+import subprocess
+import sys
+
+import pytest
+from git import Repo
+
+PRIMER_DIRECTORY = ".pylint_primer_tests"
+
+
+@contextlib.contextmanager
+def _patch_stdout(out):
+    sys.stdout = out
+    try:
+        yield
+    finally:
+        sys.stdout = sys.__stdout__
+
+
+def lazy_git_clone(git_url: str, target_directory: str) -> None:
+    if not os.path.exists(target_directory):
+        Repo.clone_from(git_url, target_directory)
```

Comment:
How does this work if an user has already cloned the repo before but new commits get added to the repo?
If the path exists it won't get updated, right?